demo/admin/src/products/generator/ProductsGrid.cometGen.tsx

```diff
@@ -128,7 +128,7 @@ export default defineConfig<GQLProduct>({
             filterOperators: ManufacturerFilterOperators,
         },
         {
-            type: "virtual",
+            type: "manyToMany",
```

Comment:
devs might expect that type=manyToMany also takes care about renderCell and queryFields

`title` still has to be specified in some way

Reply:
> We could add a default, but it should be possible to override it in the column definition.

:+1: of course

> If we want to provide defaults, how would we do that? add a titleField option?

yes. That might even default to the first field in the relation? Alternative be required?

Reply:
Renamed the field to `labelField` and changed the type to `string`. I'm not sure if it's possible to infer the names of the relation.